pkg/vsphere/extraconfig/keys.go

```diff
@@ -27,30 +27,73 @@ import (
 )
 
 const (
-	// DefaultTagName value
+	// GuestInfoPrefix is dictated by vSphere
+	GuestInfoPrefix = "guestinfo."
+
+	// ScopeTag is the tag name used for declaring scopes for a field
+	ScopeTag = "scope"
+	// KeyTag is the tag name by which to override default key naming based on field name
+	KeyTag = "key"
+	// RecurseTag is the tag name with which different recursion properties are declared
+	RecurseTag = "recurse"
+
+	// HiddenScope means the key is hidden from the guest and will not have a GuestInfoPrefix
+	HiddenScope = "hidden"
```

Comment:
Should any of the instances of this string be updated to use this constant?

https://github.com/vmware/vic/blob/1387ab15f05282c466982415716e63a6f3c1591e/pkg/vsphere/extraconfig/keys_test.go#L38

https://github.com/vmware/vic/blob/1387ab15f05282c466982415716e63a6f3c1591e/pkg/vsphere/extraconfig/keys_test.go#L42

https://github.com/vmware/vic/blob/1387ab15f05282c466982415716e63a6f3c1591e/pkg/vsphere/extraconfig/keys_test.go#L50